### PR TITLE
Add doc about depending on the BCR infra

### PR DESCRIPTION
**Pull request by meteorcloudy**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -9,3 +9,15 @@ The default Bazel registry for the Bzlmod external dependency system of Bazel. I
...
 To contribute, check our [BCR  policies](docs/bcr-policies.md) and [contribution guidelines](docs/README.md).
+
+## Depending on the BCR infrastructure (and how not to)
+
+The core infrastructure of the Bazel Central Registry depends on Github and Google Cloud. Bazel users who enable Bzlmod by default depend on the BCR.
+While we try to keep the BCR infrastructure simple and reliable, we're not responsible for any damages caused by build failures due to potential BCR infrastructure failures.
+
+If you consider it's necessary, you can do the following to avoid depending on the BCR infrastructure while still making use of the information checked in the BCR.
+
+- Clone the BCR repository or mirror the content to your own infrastructure and use `--registry` option to change the default Bazel registry to your own.
```
Comment (Wyverald) [inline README.md:20]: link `--registry` to: https://bazel.build/reference/command-line-reference#flag--registry

File: README.md [hunk 2]
```diff
@@ -9,3 +9,15 @@ The default Bazel registry for the Bzlmod external dependency system of Bazel. I
 ## Contributing
 
 To contribute, check our [BCR  policies](docs/bcr-policies.md) and [contribution guidelines](docs/README.md).
+
+## Depending on the BCR infrastructure (and how not to)
+
+The core infrastructure of the Bazel Central Registry depends on Github and Google Cloud. Bazel users who enable Bzlmod by default depend on the BCR.
+While we try to keep the BCR infrastructure simple and reliable, we're not responsible for any damages caused by build failures due to potential BCR infrastructure failures.
```
Comment (Wyverald) [inline README.md:16]: "we're not responsible for ..." sounds a bit bad :P something like "we assume no liability for ..." might be better

File: README.md [hunk 3]
```diff
@@ -9,3 +9,15 @@ The default Bazel registry for the Bzlmod external dependency system of Bazel. I
...
+
+If you consider it's necessary, you can do the following to avoid depending on the BCR infrastructure while still making use of the information checked in the BCR.
```
Comment (Wyverald) [inline README.md:18]: nit: consider it's necessary -> consider it necessary; checked in the BCR -> checked into the BCR

File: README.md [hunk 4]
```diff
@@ -9,3 +9,15 @@ The default Bazel registry for the Bzlmod external dependency system of Bazel. I
...
+
+- Clone the BCR repository or mirror the content to your own infrastructure and use `--registry` option to change the default Bazel registry to your own.
+- Host your own mirror for all source archive URLs and change the mirror URL in `./bazel_registry.json` to your own.
+  You can run the `./tools/print_all_src_urls.py` script to get the list of source URLs for all Bazel modules checked in the BCR.
+  For example, `https://foo/bar.zip` should be mirrored to `https://<your mirror>/foo/bar.zip`.
```
Comment (Wyverald) [inline README.md:23]: maybe use `https://foo.com/bar.zip` for a slightly more realistic example?

File: README.md (hunk 1 above)
Comment (meteorcloudy) [reply README.md]: Done.

File: README.md (hunk 2 above)
Comment (meteorcloudy) [reply README.md]: Thanks!

File: README.md (hunk 3 above)
Comment (meteorcloudy) [reply README.md]: Done

**Review by Wyverald — APPROVED** [review]

**Pull request merged**